rest/v1/include.php

```diff
@@ -16,7 +16,7 @@
 require_once("../../lib/vendor/defuse-crypto.phar");
 require_once("../../lib/functions.inc.php");
 if ($use_pwnedpasswords) {
-    require_once("../../lib/vendor/ron-maxweb/pwned-passwords/src/PwnedPasswords/PwnedPasswords.php");
+    require_once("../../lib/vendor/mxrxdxn/pwned-passwords/src/PwnedPasswords/PwnedPasswords.php");
```

Comment:
Maybe in another PR, we should use composer also for webservices

Reply:
oh, and actually, the above probably won't work. As seen running Travis-CI tests, swapping one for the other would return an error regarding GruzzleHttp\Client not being loaded, which I fixed by calling the autoload.php instead, ... I'll fix this ...